lib/server/monalisa-synchronization/MonALISASynchronizer.js

```diff
@@ -100,6 +102,47 @@ class MonALISASynchronizer {
 
         return Object.fromEntries(lastRunNumbersEntries);
     }
+
+    /**
+     * Synchronize with data from MonALISA
+     * Update Simulation Passes data in local DB
+     * @return {Promise<void>} promise
+     */
+    async synchronizeSimulationPassesFromMonALISA() {
+        try {
+            const simulationPasses = await this.monALISAInterface.getSimulationPasses();
+
+            // eslint-disable-next-line require-jsdoc
+            const transactionablePipeline = async (simulationPassData) => {
```

Comment:
Please extract a function with a proper name.
The function should not be named after the context in which you expect it to be used, but after what it does!
And again, stop disabling jsdoc!!!!

Reply:
So review more carefully: https://github.com/AliceO2Group/Bookkeeping/blob/a6d9a8ab8dee9977f86c07e61eb1f8cb9154be5c/lib/server/monalisa-synchronization/MonALISASynchronizer.js#L46

Reply:
function extracted